test/kpod_push.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
+
+function teardown() {
+    cleanup_test
+}
+
+@test "kpod push to containers/storage" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull "$IMAGE"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS push "$IMAGE" containers-storage:[$ROOT]busybox:test
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run crioctl image remove "$IMAGE"
+    run crioctl image remove busybox:test
+    stop_crio
+}
+
+@test "kpod push to directory" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull "$IMAGE"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run mkdir /tmp/busybox
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS push "$IMAGE" dir:/tmp/busybox
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run crioctl image remove "$IMAGE"
+    run rm -rf /tmp/busybox
```

Comment:
Better to not use the `run` wrapper here if we're not going to check the result.